### ISSUE TITLE
Add Ability to Sort Any Start Menu Folder by Last Access/Most Recently Used and set the # of Items.

### ISSUE THREAD

**Issue by memorialparker** [Open-Shell/Open-Shell-Menu#194]
In particular, the folders
Recent Programs - sorts by most recently used but will only show the top 25 items.
Recent Documents - seems to work but includes all types of image files along with Documents. 
Recent Folders - always sorts alphabetically no matter what I do and limited to 30 items. 
Recent Items - always sorts alphabetically but allows many many items. Think I have 100.
Recent Items Instance Folder - alphabetical sort and limited to 20 items. 

I have a feeling Windows 10 keeps some type of master MRU list. They for various Recent's folders, it somehow filters the list to only Docs, exe's, ... depending on what they want to show.

If there are Windows 10 registry settings that control any of the above, please let me know.

**Comment by timh-86** [comment]
I agree. There is a "recent items" as a default option, why not a "recent folders"? 

I added it manually, but as memorialparker states, it is impossible to sort it by "most recently accessed". And since the list is so long, it becomes a mess and hence useless.

Would be much appreciated if you added this feature, Thanks!

**Comment by glowingwire** [comment]
I personally, and for clients, disable most things that change without user
interaction.

Having those in their own contained areas is ok.  I would like the list of
files and folders to have full paths available.

Also, a wizard to help people organize their start menus would be awesome.

People I know could choose from most frequently used websites, and apps...
This is how I configure the quicklaunch bar, and the items in the
startmenu.
